sql/moz-fx-data-shared-prod/firefox_ios_derived/retention_v1/metadata.yaml

```diff
@@ -0,0 +1,37 @@
+friendly_name: Client Retention - Firefox iOS (Aggregated)
+description: |
+  Client Retention (Firefox iOS) metrics aggregated on:
+  - metric_date
+  - first_seen_date
+  - app_name
+  - channel
+  - country
+  - adjust_ad_group
+  - adjust_campaign
+  - adjust_creative
+  - adjust_network
+
```

Comment:
+ suspicious device

Reply:
@bani oh, I thought in the aggregation we wanted to exclude it completely?